test/ui/widgets/grade_button_test.dart

```diff
@@ -101,5 +108,38 @@ void main() {
                 'so "N/A" should be displayed');
       });
     });
+
+    group('navigate when tapped to grade button - ', () {
+      testWidgets('Display acronym of the course and the current grade ', (WidgetTester tester) async {
```

Comment:
Il faudrait aussi mettre à jour le nom du test, quelque chose comme "Grade button redirects to grades view when tapped"